src/core/tests/type_prop/istft.cpp

```diff
@@ -0,0 +1,427 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/istft.hpp"
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/subtract.hpp"
+
+namespace ov {
+namespace test {
```

Comment:
```suggestion
namespace ov::test {
```
Usually less `{` in code

Reply:
Done